app/models/auction_query.rb

```diff
@@ -98,7 +98,7 @@ def public_find(id)
       .find(id)
   end
 
-  def my_bids(user_id)
+  def user_has_bid(user_id)
```

Comment:
yeah that is tricky....what about something like `with_bid_for_user(user_id)` too wordy?